### ISSUE TITLE
[1.18.2] CTD after updating to 1.0.1

### ISSUE THREAD

**Issue by drakray** [AmarokIce/PineappleDelight#1]
https://pastebin.com/N72ALnth

Failure message: Pineapple Delight (pineapple_delight) encountered an error during the common_setup event phase
java.lang.NullPointerException: Registry Object not present: pineapple_delight:pineapple_crop

<details><summary>Snipet</summary>

```
A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at java.util.Objects.requireNonNull(Objects.java:334) ~[?:?] {re:mixin}
-- MOD pineapple_delight --
Details:
	Mod File: /C:/Users/drakr/AppData/Roaming/.minecraft/1182.1/mods/pineappledelight-1.18.2-1.0.1.jar
	Failure message: Pineapple Delight (pineapple_delight) encountered an error during the common_setup event phase
		java.lang.NullPointerException: Registry Object not present: pineapple_delight:pineapple_crop
	Mod Version: 1.0
	Mod Issue URL: NOT PROVIDED
	Exception message: java.lang.NullPointerException: Registry Object not present: pineapple_delight:pineapple_crop
Stacktrace:
	at java.util.Objects.requireNonNull(Objects.java:334) ~[?:?] {re:mixin}
	at net.minecraftforge.registries.RegistryObject.get(RegistryObject.java:320) ~[forge-1.18.2-40.1.84-universal.jar%23371!/:?] {re:mixin,xf:fml:forge:registry_object_binary_compat,re:classloading,xf:fml:forge:registry_object_binary_compat}
	at club.someoneice.pineapple.init.BlockList.lambda$static$0(BlockList.java:16) ~[pineappledelight-1.18.2-1.0.1.jar%23308!/:1.0] {re:classloading}
```

</details>

**Comment by AmarokIce** [comment]
Sorry I'm late!
Please check if the issue still exists in the latest version

**Comment by drakray** [comment]
Hi, I think it's fixed?, if I recall last time I played 1.18.2 it was working
